utils/testrunner/Matchers.js

```diff
@@ -43,50 +43,60 @@ class Expect {
 
     function applyMatcher(matcherName, matcher, inverse, value, ...args) {
       const result = matcher.call(null, value, ...args);
-      const message = result.message || `expect.${matcherName} failed`;
+      const message = `expect.${matcherName} failed` + (result.message ? `: ${result.message}` : '');
       console.assert(result.pass !== inverse, message);
     }
   }
 }
 
 const DefaultMatchers = {
   toBe: function(value, other, message) {
+    message = message || `${value} == ${other}`;
     return { pass: value === other, message };
   },
 
   toBeFalsy: function(value, message) {
+    message = message || `${value}`;
     return { pass: !value, message };
   },
 
   toBeTruthy: function(value, message) {
+    message = message || `${value}`;
     return { pass: !!value, message };
   },
 
   toBeGreaterThan: function(value, other, message) {
+    message = message || `${value} > ${other}`;
     return { pass: value > other, message };
   },
 
   toBeGreaterThanOrEqual: function(value, other, message) {
+    message = message || `${value} >= ${other}`;
     return { pass: value >= other, message };
   },
 
   toBeLessThan: function(value, other, message) {
+    message = message || `${value} < ${other}`;
     return { pass: value < other, message };
   },
 
   toBeLessThanOrEqual: function(value, other, message) {
+    message = message || `${value} <= ${other}`;
     return { pass: value <= other, message };
   },
 
   toBeNull: function(value, message) {
+    message = message || `${value} == null`;
     return { pass: value === null, message };
   },
 
   toContain: function(value, other, message) {
+    message = message || `${value} ⊇ ${other}`;
     return { pass: value.includes(other), message };
   },
 
   toEqual: function(value, other, message) {
+    message = message || `Object::${value.constructor.name} ≈ Object::${other.constructor.name}`;
```

Comment:
Jasmine outputs json here. We use toEqual with plane objects, so the constructor name doesn't help much.

Reply:
I remember huge JSON.stringifed output of `Request` objects in our tests from time to time which I hated, but it seems to be never a case for `toEqual` matcher. Switched to `JSON.stringify` here.